cmd/queue/main.go

```diff
@@ -217,6 +231,44 @@ func main() {
 		logger.Fatalw("Failed to parse localhost URL", zap.Error(err))
 	}
 
+	queueProxyL3 := fmt.Sprintf("%s:%d", servingRevision, networking.ServiceHTTPPort)
+	zipkinEndpoint, err := zipkin.NewEndpoint(servingRevision, queueProxyL3)
+
+	if err != nil {
+		logger.Error("Unable to create tracing endpoint")
+		return
+	}
+
+	oct := tracing.NewOpenCensusTracer(
+		tracing.WithZipkinExporter(tracing.CreateZipkinReporter, zipkinEndpoint),
+	)
+	fmt.Println("TRACING VARS")
+	fmt.Println(os.Getenv("TRACING_CONFIG_ENABLE"))
+	fmt.Println(os.Getenv("TRACING_CONFIG_DEBUG"))
+	fmt.Println(os.Getenv("TRACING_CONFIG_SAMPLE_RATE"))
+	fmt.Println(os.Getenv("TRACING_CONFIG_ZIPKIN_ENDPOINT"))
+
+	enableVal, err := strconv.ParseBool(os.Getenv("TRACING_CONFIG_ENABLE"))
+	if err != nil {
+		logger.Errorw("Error getting tracing enable var", zap.Error(err))
```

Comment:
it might be simply unset, so logging an error seems like an overkill.

Reply:
That makes sense - I ended up logging Info level and setting a default value if it's unset.